README.md

```diff
@@ -1,10 +1,12 @@
 
+
 # AppCompS Team | [Coursework Breif](https://vle.dmu.ac.uk/bbcswebdav/pid-5687216-dt-content-rid-11305925_1/courses/CTEC2902_2122_520/CTEC2902_Assignment_2_2021-2022%281%29.pdf) [<img src="https://user-images.githubusercontent.com/41834061/155518050-bc9db224-532c-487f-a538-a074905e6bd7.svg" width="30" height="30">](https://github.com/DMU-CTEC2902-2022/AppCompS_Team/wiki) 
 
 
 
 
 
+
 ## Submissions
```

Comment:
I think having it just in Wiki works perfectly fine. _Repetition_? 
@No3Mc @Talhamemon25

Reply:
Rather than going to wiki to look it up. I find it easier to just get an instant reminder bro. I think for now we shall leave it as it is.

Reply:
@Hassaan2612 exactly but tbh I do agree with @Talhamemon25  idea too as we will find instant readme easier compared to wiki. Currently, I think just update your wikis and let the readme be as it is. Just slight changes to it. Later on we will move that to Wiki and just edit the readme accordingly.